crawler/src/main/java/com/movie/recommender/crawler/client/UrlBuilder.java

```diff
@@ -19,4 +20,13 @@ public static String movieKeywordsUrl(String movieId) {
         return BASE_TMDB_URL + "/movie/" + movieId + "/keywords";
     }
 
+    public static String movieShowtimesUrl(String movieName,String location, String language) {
+        return BASE_SERPAPI_URL + "search.json?q="+replaceSpaces(movieName)+ "+showtimes&location="
+                + replaceSpaces(location) + "&hl=" + language ;
+    }
+
+    private static String replaceSpaces(String input) {
```

Comment:
instead of using `replaceSpaces` to encode spaces as +, use URLEncoder.encode

```
private static String encode(String input) {
        return URLEncoder.encode(input, StandardCharsets.UTF_8);
    }
 ```